crates/rspack_binding_options/src/options/raw_split_chunks.rs

```diff
@@ -24,7 +24,7 @@ pub struct RawSplitChunksOptions {
   #[napi(ts_type = "RegExp | 'async' | 'initial' | 'all'")]
   #[derivative(Debug = "ignore")]
   pub chunks: Option<Chunks>,
-  //   pub automatic_name_delimiter: String,
+  pub automatic_name_delimiter: Option<String>,
```

Comment:
since the default value is assigned on js side, this will never be None, this can change to `String`